custom_components/tuya_local/devices/aktobis_wdh_310ek_dehumidifier.yaml

```diff
@@ -0,0 +1,175 @@
+name: Aktobis WDH-310EK dehumidifier
+products:
+  - id: bf43c599b40eada03aivfh
+    name: WDH-310EK
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
+        - dps_val: false
+          icon: "mdi:air-humidifier-off"
+          icon_priority: 2
+        - dps_val: true
+          icon: "mdi:air-humidifier"
+          icon_priority: 4
+    - id: 3
+      name: humidity
+      type: integer
+      range:
+        min: 30
+        max: 70
+      mapping:
+        - step: 5
+    - id: 6
+      name: current_humidity
+      type: integer
+      range:
```

Comment:
Range is not needed for read-only values (unless you are inverting them or translating to a different target_range).